src/components/QuranReader/TajweedBar/TajweedBar.module.scss

```diff
@@ -0,0 +1,125 @@
+@use "src/styles/breakpoints";
```

Comment:
The Tajweed bar gets pushed down too much which leaves a gap between the progress bar and the Tajweed bar only when the navbar is hiding 

<img width="1728" alt="Screenshot 2024-06-23 at 9 36 54 PM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/231c4775-53c2-455a-873f-3e8b7a2ba7e6">